pkg/controller/operators/olm/apiservices.go

```diff
@@ -378,50 +424,67 @@ func (a *Operator) installAPIServiceRequirements(desc v1alpha1.APIServiceDescrip
 		depSpec.Template.Spec.Containers[i] = container
 	}
 
-	// create APIService with fresh CA bundle
-	caCertPEM, _, err := ca.ToPEM()
-	if err != nil {
-		logger.Debugf("unable to convert CA certificate to PEM format for APIService %s", apiServiceName)
-		return nil, err
+	// changing depSpec.Template's name to force a rollout
+	// ensures that the new secret is used by the apiserver if not hot reloading
+	podTemplateName := names.SimpleNameGenerator.GenerateName(apiServiceName)
```

Comment:
Is there a safer way to rollout? It's unlikely but there may be dependencies on the pod names. 

Does updating metadata trigger the rollout? Could we add a short hash of the cert as an annotation or something?

Reply:
[Yes, it does ](https://github.com/kubernetes/kubernetes/issues/27081#issuecomment-229252547) - will do